Gemfile

```diff
@@ -15,6 +15,9 @@ gem "puma", "~> 5.0"
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem "tzinfo-data", platforms: %i[mingw mswin x64_mingw jruby]
 
+# For compiling our frontend assets
+gem "vite_rails"
```

Comment:
Should we version this? I'm not sure I have super strong feelings but my instinct is yes.